code/modules/materials/definitions/metals/copper.dm

```diff
@@ -17,3 +17,5 @@
 	refraction = MATERIAL_RESISTANCE_NONE
 	absorption = MATERIAL_RESISTANCE_HIGH
 	nullification = MATERIAL_RESISTANCE_NONE
+
+	material_constraints = MATERIAL_CONSTRAINT_RIGID | MATERIAL_CONSTRAINT_CONDUCTIVE
```

Comment:
i don't really do irl materials science (yet) 

is this conductive and the others not or is this just more conductive?
we should have conductivity as a variable if it's the latter case (`p_conductivity` or something, the `p_` denotes a material physical property)